orchestration/runner.py

```
@@ -0,0 +1,391 @@
+        help="Resume existing session for this directory",
+    )
+    run_parser.add_argument(
+        "--worktree", dest="worktree", action="store_true", default=False,
+        help="Create a git worktree first (only with --async)",
+    )
+
+    # Add live, list, attach, kill subcommands
+    live_parser = subparsers.add_parser("live", help="Start live orchestration mode")
+    live_parser.add_argument("--session", "-s", help="Session name to use")
+
+    list_parser = subparsers.add_parser("list", help="List active sessions")
```

Ruff (0.15.2)

[error] Local variable `list_parser` is assigned to but never used

Remove assignment to unused variable `list_parser`

(F841)